### DISCUSSION TITLE
SyncUps Project won't load

### DISCUSSION THREAD

**Discussion by easiwriter** [pointfreeco/swift-composable-architecture#3336] [General]
I have tried several times to open the SyncUps project but it just hangs. This happens in Xcode 15.4 and the latest version 16 Beta. Anyone else had this happen? Also Xcode asks for permission to access data from other apps. Why? I've never seen this before.

**Comment by mbrandonw** [comment]
Hi @easiwriter, the project opens just fine for me and runs:

<img width="1298" alt="image" src="https://github.com/user-attachments/assets/73ae0590-bb6e-4a63-af0d-356ff0c1e7b9">

There is nothing special about the project, and so if it is behaving erratically there may be something wrong with your Xcode install.

**Comment by easiwriter** [comment]
Thankd for the reply. It doesn't load on 15.4 or the latest beta. I will try reinstalling Xcode. CheersKeith (Easiwriter)Sent from my iPadOn 31 Aug 2024, at 20:04, Brandon Williams ***@***.***> wrote:﻿
Hi @easiwriter, the project opens just fine for me and runs:
image.png (view on web)
There is nothing special about the project, and so if it is behaving erratically there may be something wrong with your Xcode install.

**Reply by easiwriter** [reply]
I originally downloaded all the code from GitHub and dragged SyncUps out of the Examples folder. It wouldn't load. I have just tried opening the project from inside the Examples folder and it worked OK. So I tried dragging it out again and, again, it wouldn't load. I have no idea why this happens.

**Reply by mbrandonw** [reply]
Hi @easiwriter, you cannot drag the SyncUps project out of the Examples directory. It uses the library code in its parent directory. Is there a reason you need to move it out? If you really need to do this, then you will need to add an explicit dependency on the Composable Architecture.

**Comment by easiwriter** [comment]
Hi Brandon,
Thanks for the explanation. The only reason I moved it was because I didn't need the rest of the Examples directory. I am happy to leave it where it is.
Regards
Keith
